static/styles/settings.css

```diff
@@ -755,9 +755,9 @@ input[type=checkbox].inline-block {
 }
 
 #settings_page .content-wrapper {
-    position: relative;
-    float: left;
-    width: calc(100% - 250px - 2px);
+    position: absolute;
+    left: 251px;
+    width: calc(100% - 250px - 1px);
```

Comment:
I usually do this to indicate that these are from two different measures:

1. 250px wide for the sidebar.
2. 1px wide for the border which is not set as `box-sizing: border-box`.

Should I add a comment?